scripts/get_web_error.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+"""
+% `python3 get_web_error.py 2023-02-03/011556535629.html | lynx --stdin`
+
+To see the contents of an Open Library user-reported like 2023-02-03/011556535629.html:
+1. Copy this script into your user directory on any Open Library host.
+2. Type the command above substituting in the error code you are looking for.
+
+This script will ssh into hosts ol-web1 and ol-web2 and if that file is found in the
+docker_container openlibrary_web_1, it will print its contents to stdout.
+
+Type `yes` if ssh prompts you to add the hosts to your known_hosts file.
+
+If lynx is not yet installed, please type `sudo apt-get install lynx` to install it.
+"""
+
+import subprocess
+import sys
+from collections.abc import Iterable
+
+usage = f"""Usage: {sys.argv[0]} [filename] | lynx --stdin
+filename is like 2023-02-03/000008077313.html"""
+
+
+def get_web_error(
+    filename: str,
+    hosts: Iterable[str] = ("ol-web1.us.archive.org", "ol-web2.us.archive.org"),
```

Comment:
This line is throwing an error when I run the script:
```
Traceback (most recent call last):
  File "get_web_error.py", line 28, in <module>
    hosts: Iterable[str] = ("ol-web1.us.archive.org", "ol-web2.us.archive.org"),
TypeError: 'ABCMeta' object is not subscriptable
```

Everything worked properly after I removed the type hint.

Reply:
Maybe it's best if we roll-back to the previous commit.  I think that @RayBB was right: we can simply use `docker cp` in lower-level environments.

Reply:
Dropped the type hint.